templates/plugin.js

```diff
@@ -1,3 +1,8 @@
-export default async function({ router, store }) {
+/**
+ * @type {import('@nuxt/types').Plugin}
+ */
+const plugin = async function ({ app: { router }, store }, inject) {
   //
 }
+
+export default plugin
```

Comment:
Do we need this for jsdoc?

Reply:
Yes, unfortunately.

Reply:
Not quite sure why I thought that wouldn't work :thinking: Anyway, fixed.